.github/workflows/static-binary.yaml

```diff
@@ -29,6 +31,18 @@ jobs:
       if: github.event_name != 'repository_dispatch'
       uses: actions/checkout@v1
 
+    - name: Setup Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.7'
+
+    - name: Configure GCloud Credentials
+      uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+      with:
+        version: '275.0.0'
+        service_account_email: ${{ secrets.GCP_SA_EMAIL }}
+        service_account_key: ${{ secrets.GCP_SA_KEY }}
+
```

Comment:
Can we move this whole block down? We don't need to initialize the GCS stuff if the build fails.

Reply:
Moved the block.